src/components/staking/DelegationsOverview.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div>
+    <div v-if="delegation.loaded && yourValidators.length > 0">
+      <TableValidators
+        :validators="yourValidators"
+        show-on-mobile="expectedReturns"
+      />
+    </div>
+    <TmDataMsg
+      v-else-if="yourValidators.length === 0"
+      icon="sentiment_dissatisfied"
+    >
+      <div slot="title">
+        No validators in your portfolio
+      </div>
+      <div slot="subtitle">
+        Head over to the
+        <router-link :to="{ name: 'Validators' }"> validator list </router-link
```

Comment:
Is there a reason to use the object syntax and not just /validators?

Reply:
In theory to decouple the route from the actual url. But let's standardize.